scripts/download_checkpoints.bat

```diff
@@ -0,0 +1,2 @@
+mkdir ..\checkpoints

```

Comment:
Please add a carriage return at the end of line 2 to avoid the ⛔ from the GitHub editor.